shared/common-adapters/list-item.desktop.js

```diff
@@ -6,9 +6,9 @@ import type {Props} from './list-item'
 
 export default class ListItem extends Component<void, Props, void> {
   render () {
-    const clickable = this.props.clickable === undefined ? true : !!this.props.clickable
+    const clickable = !!this.props.onClick
```

Comment:
then those things would have their own clickable styles and onclick handlers, and that will work as before. This is just to assign the clickable style to the whole item if there is an onClick attached.